devsetup/edpm/edpm-play.yaml

```diff
@@ -0,0 +1,217 @@
+
+                  # 99-standalone-vars
+                  ctlplane_dns_nameservers:
+                  - 192.168.122.1
+                  dns_search_domains: []
+                  tripleo_nova_compute_DEFAULT_transport_url: rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.localdomain:5672/?ssl=0
+                  tripleo_nova_compute_cache_memcache_servers: standalone.ctlplane.localdomain:11211
+                  tripleo_nova_compute_cinder_auth_url: http://192.168.24.3:5000/v3
+                  tripleo_nova_compute_cinder_password: ip3VDO3mq6JObAQqfekzt85kp
+                  tripleo_nova_compute_config_overrides:
+                    DEFAULT:
+                      oslo_messaging_notifications_transport_url: rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.localdomain:5672/?ssl=0
+                      transport_url: rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.localdomain:5672/?ssl=0
+                    cache:
+                      memcache_servers: standalone.ctlplane.localdomain:11211
+                    cinder:
+                      auth_url: http://192.168.24.3:5000/v3
+                      password: ip3VDO3mq6JObAQqfekzt85kp
+                    neutron:
+                      auth_type: v3password
+                      auth_url: http://192.168.24.3:5000/v3
+                      password: viaNg9cyAqS8N1ydQe7n3XkuM
+                      project_domain_name: Default
+                      project_name: service
+                      region_name: regionOne
+                      user_domain_name: Default
+                      username: neutron
+                    placement:
+                      auth_type: password
+                      auth_url: http://192.168.24.3:5000
+                      password: Tnqq58zC6mD9O2jaC8xfiyt3M
+                      project_domain_name: Default
+                      project_name: service
+                      region_name: regionOne
+                      user_domain_name: Default
+                      username: placement
+                      valid_interfaces: internal
+                    service_user:
+                      auth_type: password
+                      auth_url: http://192.168.24.3:5000
+                      password: 59CnvTwMZcJfxCF9NU8bGWrcd
+                      project_domain_name: Default
+                      project_name: service
+                      region_name: regionOne
+                      send_service_user_token: 'True'
+                      user_domain_name: Default
+                      username: nova
+                    vnc:
+                      novncproxy_base_url: http://192.168.24.3:6080
+                  tripleo_nova_compute_neutron_auth_type: v3password
+                  tripleo_nova_compute_neutron_auth_url: http://192.168.24.3:5000/v3
+                  tripleo_nova_compute_neutron_password: viaNg9cyAqS8N1ydQe7n3XkuM
+                  tripleo_nova_compute_neutron_project_domain_name: Default
```

Comment:
would it be reasonable to file a Jira to rename all these parameters to *not* use tripleo_ in the names?

Reply:
Yes, we're going to be migrating the ansible code from tripleo-ansible to edpm-ansible (under openstack-k8s-operators ideally). As part of that migration, all the tripleo naming/branding will be removed.